### PR TITLE
Adds a very basic round timer system to base_id.lua

### PR DESCRIPTION
**Pull request by alaswell**
Also updates ksour.lua and napoli.lua to utilize it...

4 minutes per round.

When the match begins, the timer begins to count down until either the attacking team caps or the round time limit runs out (reaches zero). If they cap, the timer is reset to 4min again. However, If the time limit runs out before the attacking team can make any progress, the teams are switched and everything resets. 

contributes to fortressforever/fortressforever#60

### CODE REVIEW

**Comment by squeek502** [comment]
So something that wasn't addressed in fortressforever/fortressforever#60 but needs to be fixed in my opinion is that the scoring in ID feels (or is) really arbitrary, and it's unclear why a certain team is winning or wins in the end.

This might need to be addressed by giving the defending team points if they successfully hold a capture point, or something like that. If the number of rounds don't really correspond with when the map ends, then it could end in a weird spot that makes it unfair for one team (one team could get an extra chance at offense, etc).

Basically, making the winning team always be the team that deserves to win is the other problem that needs to be solved in the ID Lua.

**Comment by alaswell** [comment]
I totally agree, and I actually looked into all of this when I was considering giving the D points for a hold, but ultimately decided that it wasn't really fair to give them 2 points every 30 seconds and then give them a huge lump sum of points at the end of the round timer too, while O only gets points for capping. Haha and I wasn't sure if it should be it's own issue or not so I left it alone.
